.gitignore

```diff
@@ -1,2 +1,4 @@
 __pycache__
 models/
+samples
```

Comment:
i don't think this needs to be in `.gitignore`, i could update the README to tee to `/tmp/` or something

Reply:
I'll remove the .gitignore changes then.